public/src/components/admin/Admin.tsx

```diff
@@ -22,11 +114,31 @@ export class Admin extends React.Component<AdminProps, {}> {
         }
     }
     render() {
+
+        
         return (
-            <div>
-                <h1>ADMIN PANEL</h1>
-                <span>dsaaaaaaaaaaaaaaaaaaaaaaaa</span><button onClick = {(e) => { this.purchaseFromExchange(e) }}>Send</button>
-            </div>
+            <React.Fragment>
+                <div className="adminPanel">
+                  <h1>ADMIN PANEL</h1>
+                  <span>dsaaaaaaaaaaaaaaaaaaaaaaaa</span><button onClick = {(e) => { this.purchaseFromExchange(e) }}>Send</button>       
+                </div>
+
+                {/* Dividend Feature */}
+                
+                <div className="dividendPanel" style={{ float: 'right' }}>
+                  <Dividend 
+                   sessionMd={this.props.sessionMd} 
+                   stockBriefInfoMap={this.props.stockBriefInfoMap}
+                   currentStockId={this.state.currentStockId}
```

Comment:
No need to make `currentStockId` and `dividendAmount` as props, they can be state variables inside Dividend. You can simplify the functions also accordingly.